types/ibc.go

```diff
@@ -41,18 +41,30 @@ func (t IBCTimeoutBlock) IsZero() bool {
 	return t.Revision == 0 && t.Height == 0
 }
 
+type IBCTimeoutBoth struct {
+	Block     IBCTimeoutBlock `json:"block"`
+	Timestamp uint64          `json:"timestamp_nanos"` // TODO: simplify to just "timestamp" in Rust
```

Comment:
Simpler field name coming with https://github.com/CosmWasm/cosmwasm/pull/902. Timestamps are always nanoseconds in IBC and we have it documented right in the enum case.

Reply:
> Timestamps are always nanoseconds in IBC and we have it documented right in the enum case.

Let's discuss this in https://github.com/CosmWasm/cosmwasm/pull/902
I wrote much of this code and twice returned seconds there, as there were no helpers, I just took `env.block.time + 2000` or such.

Reply:
Happy for simpler name in Go

Reply:
Okay, we can stick to it if helps. No rush here. I just felt it was overly long when writing the glue code.